apps/judicial-system/backend/src/app/modules/case-table/caseTable.cellGenerators.ts

```diff
@@ -300,46 +330,48 @@ const defendants: CaseTableCellGenerator = {
       separate: true,
     },
   },
-  generate: (c: Case): StringGroupValue | undefined =>
+  generate: (c: Case): CaseTableCell<StringGroupValue> =>
     c.defendants && c.defendants.length > 0
-      ? {
-          s: [
+      ? generateCell({
+          strList: [
```

Comment:
Same formatting here as above 👀

Reply:
This is slightly different, the + is on a separate line here.